ambari-metrics/ambari-metrics-timelineservice/conf/unix/ambari-metrics-collector

```diff
@@ -231,7 +231,7 @@ function start()
   # hbase_daemon "zookeeper" "start"
   #	hbase_daemon "master" "start"
   #	hbase_daemon "regionserver" "start"
-  if [ !"${DISTRIBUTED_HBASE}" ]; then
```

Comment:
The script failed to check `${DISTRIBUTED_HBASE}` because a space is required between `!` and `"`

Reply:
I have tried your way, but it is not right. I think we should use it in this way: https://stackoverflow.com/questions/2953646/how-to-declare-and-use-boolean-variables-in-shell-script